src/Host/Broker/Impl/Security/SecurityManager.cs

```diff
@@ -58,24 +58,37 @@ public class SecurityManager {
             IntPtr token;
             WindowsIdentity winIdentity = null;
             string profilePath = "";
+
+            _logger.LogInformation($"Attempting login for user: {context.Username}");
             if (NativeMethods.LogonUser(user.ToString(), domain.ToString(), context.Password, (int)LogonType.LOGON32_LOGON_NETWORK, (int)LogonProvider.LOGON32_PROVIDER_DEFAULT, out token)) {
+                _logger.LogInformation($"Login succeeded for user: {context.Username}");
                 winIdentity = new WindowsIdentity(token);
                 StringBuilder profileDir = new StringBuilder(NativeMethods.MAX_PATH);
                 uint size = (uint)profileDir.Capacity;
+
+                _logger.LogInformation($"Attempting profile creation for user: {context.Username}");
                 uint error = NativeMethods.CreateProfile(winIdentity.User.Value, user.ToString(), profileDir, size);
                 // 0x800700b7 - Profile already exists.
                 if (error != 0 && error != 0x800700b7) {
+                    _logger.LogError($"Profile creation failed for user [{context.Username}] with error: {error.ToString("X")}");
```

Comment:
The messages should be consistent with respect to square brackets around username (either present everywhere, or absent everywhere).

Also, maybe single or double quotes would be a better fit? Or is [username] some sort of standard convention for Win32 usernames?